package shm_serving;

message NamedLoader {
  required string file_name = 1;
  optional bool exclusive = 2;
}

message DataServiceConfig {
  repeated NamedLoader named_loaders = 1;
}
